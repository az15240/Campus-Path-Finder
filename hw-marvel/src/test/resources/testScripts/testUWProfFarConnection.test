# Tests data set from UWProfessorCustom.csv. Adapted from https://edstem.org/us/courses/32028/discussion/2601400.
# This file tests longer paths. are short ones with maximum of 2 edges.

LoadGraph g UWProfessorsCustom.csv

ListNodes g

FindPath g Ru.Anderson R.Lin

FindPath g Schafer Grossman

FindPath g Thachuk Y.Lee

FindPath g OveisGharan Perkins