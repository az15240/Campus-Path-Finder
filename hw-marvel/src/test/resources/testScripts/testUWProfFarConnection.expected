# Tests data set from UWProfessorCustom.csv. Adapted from https://edstem.org/us/courses/32028/discussion/2601400.
# This file tests longer paths. are short ones with maximum of 2 edges.

loaded graph g

g contains: A.Rao Beame Grossman Hsia J.Lee Karlin Kohlbrenner Kohno Leang Maas OveisGharan Perkins R.Lin Ri.Anderson Roesner Ru.Anderson Schafer Tang Tessaro Thachuk Tsun Weber Wilcox Y.Lee Zatloukal

path from Ru.Anderson to R.Lin:
Ru.Anderson to Weber via 332
Weber to Tessaro via 312
Tessaro to R.Lin via 490C

path from Schafer to Grossman:
Schafer to Ru.Anderson via 391
Ru.Anderson to Hsia via 351
Hsia to Perkins via 333
Perkins to Wilcox via 331
Wilcox to Grossman via 341

path from Thachuk to Y.Lee:
Thachuk to Hsia via 333
Hsia to Ru.Anderson via 351
Ru.Anderson to Weber via 332
Weber to Y.Lee via 421

path from OveisGharan to Perkins:
OveisGharan to Weber via 421
Weber to Zatloukal via 311
Zatloukal to Perkins via 331